core/src/test/scala/unit/kafka/server/AddPartitionsToTxnManagerTest.scala

```diff
@@ -70,7 +72,7 @@ class AddPartitionsToTxnManagerTest {
   private val authenticationErrorResponse = clientResponse(null, authException = new SaslAuthenticationException(""))
   private val versionMismatchResponse = clientResponse(null, mismatchException = new UnsupportedVersionException(""))
   private val disconnectedResponse = clientResponse(null, disconnected = true)
-  private val transactionSupportedOperation = genericError
+  private val transactionSupportedOperation = addPartition
```

Comment:
Do most of the tests here do addPartition rather than the verification?

Reply:
Will revert it back and make some of the tests testing both genericError and addPartition